pkg/cluster/controllers/storage/job.go

```diff
@@ -116,6 +121,10 @@ func (m *MinIOController) generateMinIOInitJob(ctx context.Context, harborcluste
 		},
 	}
 
+	job.Spec.Template.ObjectMeta.OwnerReferences = []metav1.OwnerReference{
+		*metav1.NewControllerRef(job, batchv1.SchemeGroupVersion.WithKind("Job")),
+	}
```

Comment:
Can you please use [controllerutil.SetControllerReference()](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#SetControllerReference)
```suggestion
	controllerutil.SetControllerReference(harborcluster, job, m.Scheme)
```